vars/kibanaPipeline.groovy

```diff
@@ -86,10 +86,11 @@ def withFunctionalTestEnv(List additionalEnvs = [], Closure closure) {
 
   def kibanaPort = "61${parallelId}1"
   def esPort = "61${parallelId}2"
-  def esTransportPort = "61${parallelId}3"
-  def fleetPackageRegistryPort = "61${parallelId}4"
-  def alertingProxyPort = "61${parallelId}5"
-  def corsTestServerPort = "61${parallelId}6"
+  // Keep 6x03 open for multi-node clusters
+  def esTransportPort = "61${parallelId}4-61${parallelId}5"
+  def fleetPackageRegistryPort = "61${parallelId}6"
+  def alertingProxyPort = "61${parallelId}7"
+  def corsTestServerPort = "61${parallelId}8"
```

Comment:
This allows the test in this PR to pass, but isn't the right long term solution as it only opens up enough ports for a 2-node cluster.

I wasn't sure if it was safe to make larger changes here because of the use of `parallelId`, but ideally we'd have several (maybe up to 10?) ports reserved for both ES and transport.

I think something like this would work without breaking the parallel ID... @tylersmalley any reason we shouldn't/can't use these other ports on our CI workers?
```groovy
  def kibanaPort = "61${parallelId}1"
  def esPort = "62${parallelId}1"
  // Ports 62x2-62x9 kept open for ES nodes
  def esTransportPort = "63${parallelId}1-63${parallelId}9"
  def fleetPackageRegistryPort = "64${parallelId}1"
  def alertingProxyPort = "64{parallelId}2"
  def corsTestServerPort = "64${parallelId}3"
```

Reply:
I think the proposed changes look good to me. @brianseeders anything I am missing?

Reply:
Yep, should be okay.

Reply:
There are multiple parallel workers running on each individual machine, so the `parallelId` just ensures that each one of them gets a unique port for each thing.

Reply:
Updated with my proposed changes above (waiting to see if it passes CI as expected).